common/os.go

```diff
@@ -148,6 +148,7 @@ func WriteFileAtomic(filename string, data []byte, perm os.FileMode) error {
 	} else if n < len(data) {
 		return io.ErrShortWrite
 	}
+	f.Close()
```

Comment:
can you add a comment here explaining why we need to do that.

like "close the file before renaming it. otherwise ...."

Reply:
In the same device with the same partition type, the "rename" on linux only modify the file name, so
it will not affect the existing read and write operations. But If you "rename" it to another device, it will crash ("cross-link error").
This problem may also exist in another operating systems.